app/src/main/AndroidManifest.xml

```diff
@@ -16,10 +16,16 @@
     <activity
       android:name=".home.continueplaying.ContinuePlayingActivity"
       android:theme="@style/OppiaThemeWithoutActionBar"
-      android:screenOrientation="portrait" />
+      android:screenOrientation="portrait"
+      />
```

Comment:
Press `Option + Cmd + L` to fix formattinng

Reply:
Done